addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -40,8 +40,8 @@
                     </t>
                 </button>
                 <div class="o-autogrow"/>
-                <t t-if="chatter.hasFollowers">
-                    <div class="o_ChatterTopbar_rightSection">
+                <div class="o_ChatterTopbar_rightSection" t-att-class="{ 'o-has-flap': props.hasCloseButton }">
```

Comment:
Also, do we want to render the right section if it is empty? Would condition "hasFollowers or hasCloseButton" above it make any sense?

Reply:
Will be done with followers https://github.com/odoo-dev/odoo/pull/203/files#diff-b470e7dd9ad57b30f28073528aa8544d